<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="fileRename"/>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Attach a file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a &amp;file:</string>
     </property>
     <property name="buddy">
      <cstring>fileName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="fileSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../../resources/nicos-gui.qrc">
       <normaloff>:/open</normaloff>:/open</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>File &amp;description:</string>
     </property>
     <property name="buddy">
      <cstring>fileRename</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Re&amp;name it to:</string>
     </property>
     <property name="buddy">
      <cstring>fileRename</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="fileName"/>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="fileDesc"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileName</tabstop>
  <tabstop>fileSelect</tabstop>
  <tabstop>fileRename</tabstop>
  <tabstop>fileDesc</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../resources/nicos-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
